src/app/components_ngrx/planner-list/planner-list.component.less

```diff
@@ -36,16 +37,16 @@
     }
   }
   .f8-wi-list {
-    float: left;
+    display: flex;
     width: 100%;
-    height: 100%;
     background-color: @color-pf-white;
     &-wrap {
       position: relative;
       width: 100%;
-      // height: ~"calc(e('100% - 120px'))";
-      // to avoid overflown scroll height should be there
       overflow-y: auto;
+      .f8-planner-list-title {
+        max-width: ~"calc(e('100% - 20px'))";// lesshint-disable-line
```

Comment:
make it as ```max-width: ~'calc(e("100% - 20px"))'``` then there is no need of ```// lesshint-disable-line``` comment.

Reply:
It should work. See here https://github.com/fabric8-ui/fabric8-planner/blob/master/src/app/components_ngrx/planner-board-column/planner-board-column.component.less#L28